packages/flutter_tools/test/integration.shard/test_data/hot_reload_project.dart

```diff
@@ -44,7 +53,7 @@ class HotReloadProject extends Project {
       // hot reloading worked:
       // printHotReloadWorked();
 
-      print('((((TICK $count))))');
+      print('((((TICK \$count))))');
```

Comment:
?

Reply:
oh i see, wow. how did that not get caught before.

Reply:
what is `count` even resolving to here?!

Reply:
oh, this was previously a raw string so that was safe, but I want to conditionally embed some code so I switched it out to a regular string and escaped the count interpolation to keep the same behavior.

Reply:
aaah